job-service/src/main/java/com/hpe/caf/services/job/api/DatabaseHelperFactory.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2016-2024 Open Text.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.services.job.api;
+
+import com.hpe.caf.services.configuration.AppConfig;
+
+public class DatabaseHelperFactory {
```

Comment:
Is it possible to use Mockito to mock the constructor to avoid having to add this class and update production code?

https://www.baeldung.com/java-mockito-constructors-unit-testing#mocking-default-constructors-using-mockito